ngi_pipeline/engines/piper_ngi/launchers.py

```diff
@@ -41,9 +41,7 @@
                                           match_files_under_dir
 from ngi_pipeline.utils.classes import with_ngi_config
 from ngi_pipeline.utils.filesystem import fastq_files_under_dir, is_index_file
-from ngi_pipeline.utils.parsers import parse_lane_from_filename, \
-                                       find_fastq_read_pairs_from_dir, \
-                                       get_flowcell_id_from_dirtree
+#from ngi_pipeline.utils.parsers import get_flowcell_id_from_dirtree
```

Comment:
Are we going to something with this in the future?

Reply:
Ah, no, I removed the function but missed this line. I'll remove it now, well spotted! 😄